apps/firebase/src/celo-adapter.ts

```diff
@@ -35,7 +36,13 @@ export class CeloAdapter {
     if (this.initialized) {
       return true
     }
-    this.mento = await Mento.create(this.signer)
+    try {
+      this.mento = await Mento.create(this.signer)
+    } catch (e) {
+      // Remove block once Broker contract issues are sorted out
+      console.info('Failed to initialize Mento, using backup methods.')
+      this.useMento = false
```

Comment:
I added in this `useMento` thing so that the adapter would still ask for users to call `init`